src/Components/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import './Tabs.scss';
+
+type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
```

Comment:
add such types to separate file
you can use `react-app-env.d.ts`

Reply:
Fixed.